source/ember-cli/index.md

```diff
@@ -0,0 +1,58 @@
+Installing Ember is *easy*! And our new install process includes *Ember CLI*, Ember's build tool
```

Comment:
With versioned guides, it's no biggie- next version we can remove the new and even some of the enthusiasm too (I may have went overboard with the italics and exclamations)